### PR TITLE
chore: update packages, including resolving nested requirements to non-vulnerable versions

### PR DESCRIPTION
**Pull request by katelynsills**
Update packages, including resolving nested requirements to non-vulnerable versions

Related to https://github.com/Agoric/agoric-sdk/issues/3007

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,5 +127,9 @@
   "prettier": {
     "trailingComma": "all",
     "singleQuote": true
+  },
+  "resolutions": {
+    "postcss": "8.3.0",
```
Comment (dckc) [inline package.json:132]: Nifty!

TIL: [Selective dependency resolutions \| Yarn](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)
cc @kriskowal @warner @michaelfig

File: yarn.lock [hunk 2]
```diff
@@ -5697,30 +5869,28 @@ is-stream@^1.1.0:
   resolved "https://registry.yarnpkg.com/is-stream/-/is-stream-1.1.0.tgz#12d4a3dd4e68e0b79ceb8dbc84173ae80d91ca44"
   integrity sha1-EtSj3U5o4Lec6428hBc66A2RykQ=
 
-is-string@^1.0.5:
-  version "1.0.5"
-  resolved "https://registry.yarnpkg.com/is-string/-/is-string-1.0.5.tgz#40493ed198ef3ff477b8c7f92f644ec82a5cd3a6"
-  integrity sha512-buY6VNRjhQMiF1qWDouloZlQbRhDPCebwxSjxMjxgemYT46YMd2NR0/H+fBhEfWX4A/w9TBJ+ol+okqJKFE6vQ==
-
-is-svg@^3.0.0:
```
Comment (dckc) [inline yarn.lock:5705]: looks like `is-svg` goes away completely. cool.

**Review by dckc — APPROVED** [review]
Thanks for addressing this promptly. The `resolutions` item in `package.json` looks particularly handy!

**Comment by katelynsills** [comment]
Looks like the build is having trouble, so my pushing the envelope on what Vuepress will use may have been too far. Vuepress has not released a new version that updates the dependencies though. This needs more effort.

**Comment by katelynsills** [comment]
Ah, Vuepress v1 is no longer supported because they are working on v2: https://github.com/vuejs/vuepress/issues/2744

**Comment by katelynsills** [comment]
Closing as the resolutions forcing Vuepress to use updated dependencies break Vuepress V1. See #513 for the issue to migrate to V2

**Pull request closed without merging**